content/docs/ops/calculate-queue-capacity.md

```diff
@@ -0,0 +1,124 @@
+---
+menu:
+  docs:
+    parent: operations
+title: Calculate Queue Capacity
+---
+
+Let's outline below how to calculate our `queue` in our ELK stack. You have to
+be [logged into a staging jumpbox]({{< relref "troubleshooting-bosh.md" >}}) in
+order to get the information to perform the calculation. We calculate the
+numbers on staging as in order to ensure that we don't affect production logs or
+production deployments of Cloud Foundry. You will be turning off all the
+`parsers` in order to allow the `queue` to fill up gradually so you can
+calculate logs capacity.
+
+## Preparing the queue
+
+In order to fill up the queue, we'll need to pause the parser VMs that are
+running. Once inside the jumpbox, target the logsearch deployment when checking
+the vms.
+
+```sh
+$ bosh vms logsearch
+# ... output shortened for brevity
+Acting as user 'admin' on deployment 'logsearch' on 'bosh'
+
++----------------------+---------+-----+-----------+-------------+
+| VM                   | State   | AZ  | VM Type   | IPs         |
++----------------------+---------+-----+-----------+-------------+
+|                      | running | n/a |           |             |
+| parser/0 (yyy)       | running | n/a | parser    | 192.168.x.x |
+| parser/1 (xxx)       | running | n/a | parser    | 192.168.x.x |
+|                      | running | n/a |           |             |
++----------------------+---------+-----+-----------+-------------+
+```
+
+Once you have the name of the VMs you can run the following commands to turn off
+the parsers. Please turn them back on after the calculations.
+
+You will first need to ssh into each parser VM you find. Let's log into the
```

Comment:
Can we just simply this to running one command?

`bosh-cli -e env -d logsearch ssh parser -- sudo /var/vcap/bosh/bin/monit stop parser`